controller:
  ros__parameters:
    max_vel_ang: [0.0, 0.0, 1.67]
    max_vel_lin: [0.22, 0.0, 0.0] 
    freq: 5.0 
    max_angle: [1.57, 4.71] 
    threshold: 0.5
    goal: [6.5, 3.0, 0.5]
